src/components/Toggle.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { View, Switch } from 'react-native';
+import { Icon } from 'react-native-elements';
+
+interface ToggleProps {
```

Comment:
Since this is the only component being defined in this file, we can just call this `Props`

```suggestion
interface Props {
```